homeassistant/components/climate/toon.py

```diff
@@ -0,0 +1,89 @@
+"""
+Toon van Eneco Thermostat Support.
+
+This provides a component for the rebranded Quby thermostat as provided by
+Eneco.
+"""
+
+from homeassistant.components.climate import (ClimateDevice, ATTR_TEMPERATURE)
+from homeassistant.const import TEMP_CELSIUS
+
+import homeassistant.components.toon as toon_main
+
+STATE_HEAT = 'Comfort'
```

Comment:
Remove these as per https://github.com/home-assistant/home-assistant/pull/7527#discussion_r115782495

This cannot be merged without this issue being addressed.